File: algo/src/test/java/com/hartwig/actin/algo/evaluation/pregnancy/CanUseAdequateAnticonceptionTest.java
package com.hartwig.actin.algo.evaluation.pregnancy;

import static com.hartwig.actin.algo.evaluation.EvaluationAssert.assertEvaluation;

import com.hartwig.actin.TestDataFactory;
import com.hartwig.actin.algo.datamodel.EvaluationResult;

import org.junit.Test;

public class CanUseAdequateAnticonceptionTest {

    @Test
    public void canEvaluate() {
        CanUseAdequateAnticonception function = new CanUseAdequateAnticonception();

        assertEvaluation(EvaluationResult.NOT_EVALUATED, function.evaluate(TestDataFactory.createMinimalTestPatientRecord()));
    }
}

File: algo/src/test/kotlin/com/hartwig/actin/algo/evaluation/reproduction/CanUseAdequateAnticonceptionTest.kt
package com.hartwig.actin.algo.evaluation.reproduction

import com.hartwig.actin.TestDataFactory
import com.hartwig.actin.algo.datamodel.EvaluationResult
import com.hartwig.actin.algo.evaluation.EvaluationAssert.assertEvaluation
import org.junit.Test

class CanUseAdequateAnticonceptionTest {
    @Test
    fun canEvaluate() {
        val function = CanUseAdequateAnticonception()
        assertEvaluation(EvaluationResult.NOT_EVALUATED, function.evaluate(TestDataFactory.createMinimalTestPatientRecord()))
    }
}